components.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
+  "style": "default",
+  "rsc": true,
+  "tsx": true,
+  "tailwind": {
+    "config": "tailwind.config.ts",
+    "css": "src/app/globals.css",
+    "baseColor": "gray",
+    "cssVariables": true,
+    "prefix": ""
+  },
+  "aliases": {
+    "components": "@/components",
+    "utils": "@/libs/utils"
+  }
+}
```

Comment:
Is there an ESLint rule that checks for missing newlines at the end of files, or is that under the "Fix all `eslint` errors" TODO you mentioned?

Reply:
There is, and theoratically running `yarn run lint` should show all linting errors including missing newlines